lms/urls.py

```diff
@@ -32,12 +32,12 @@
     url(r'^login_ajax/(?P<error>[^/]*)$', 'student.views.login_user'),
 
     url(r'^email_confirm/(?P<key>[^/]*)$', 'student.views.confirm_email_change'),
-    url(r'^event$', 'track.views.user_track'),
-    url(r'^performance$', 'performance.views.performance_log'),
-    url(r'^segmentio/event$', 'track.views.segmentio.segmentio_event'),
 
-    # TODO: Is this used anymore? What is STATIC_GRAB?
-    url(r'^t/(?P<template>[^/]*)$', 'static_template_view.views.index'),
+    url(r'', include('track.urls')),
+
+    url(r'', include('performance.urls')),
```

Comment:
@andy-armstrong: I added comments.  I think re-ordering should be done in a separate commit after more clean-up, which should involve a larger re-ordering.